File: pay/src/main/java/com/truenine/component/pay/autoconfig/AutoConfigEntrance.java
package com.truenine.component.pay.autoconfig;

import com.truenine.component.pay.properties.WeChatProperties;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.context.annotation.Import;

@Import(WeChatPayAutoConfiguration.class)
@EnableConfigurationProperties(WeChatProperties.class)
public class AutoConfigEntrance {


}

File: data-common/data-common-crawler/src/main/kotlin/com/truenine/component/data/common/crawler/autoconfig/AutoConfigEntrance.kt
package com.truenine.component.data.common.crawler.autoconfig

import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.context.annotation.Import

@EnableConfigurationProperties(CrawlerProperties::class)
@Import(CrawlerLauncherAutoConfig::class)
class AutoConfigEntrance 